### PR TITLE
Create 0v90.json

### PR DESCRIPTION
**Pull request by ghost**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/0v90.json
Comment (CuteDog5695) [inline domains/0v90.json:1]: File doesn't even have anything

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/0v90.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  }
+}
+
+
+{
+  "record": {
+    "A": [
+      "35.200.171.190",
+      "35.240.84.96",
+      "34.132.146.171"
```
Comment (CuteDog5695) [inline domains/0v90.json:14]: Since these are just saying "webserver is functioning normally", I'd like you to give a preview of your website

**Review by VaibhavSys — DISMISSED** [review]
There should only be one root block
Refer: https://www.is-a.dev/docs/domain-structure/#example-json-file

File: domains/0v90.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "owner": {
+    "username": "0v90",
+    "email": "0x0jx@proton.me"
+  }
+}
+
+
+{
```
Comment (CuteDog5695) [inline domains/0v90.json:9]:
```suggestion
  },
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems fine to me now

**Pull request merged**